test/screens/v2/screen_audio_data_test.exs

```diff
@@ -1,11 +1,79 @@
 defmodule Screens.V2.ScreenAudioDataTest do
   use ExUnit.Case, async: true
 
+  alias Screens.Config.{Screen, V2}
   alias Screens.V2.ScreenAudioData
   alias Screens.V2.WidgetInstance.MockWidget
 
+  setup do
+    %{
+      config_no_audio: %Screen{
+        app_params: %V2.BusShelter{
+          departures: %V2.Departures{sections: []},
+          footer: %V2.Footer{},
+          header: %V2.Header.CurrentStopId{stop_id: "1"},
+          alerts: %V2.Alerts{stop_id: "1"}
+        },
+        vendor: :lg_mri,
+        device_id: "TEST",
+        name: "TEST",
+        app_id: :bus_shelter_v2
+      },
+      config_audio_inactive: %Screen{
+        app_params: %V2.BusShelter{
+          audio: %V2.Audio{
+            start_time: ~U[2021-02-01T00:00:00Z],
+            stop_time: ~U[2021-02-01T01:00:00Z],
```

Comment:
Use `~T` for a time-only value (no date or timezone), here and elsewhere in the setup data.